tests/test_store.py

```diff
@@ -92,9 +94,25 @@ def test_init(self):
 
 
 class TestSynapseStorage:
-    def test_init(self, synapse_store):
+    def test_init(self, synapse_store:SynapseStorage) -> None:
         assert synapse_store.storageFileview == "syn23643253"
         assert isinstance(synapse_store.storageFileviewTable, pd.DataFrame)
+        assert synapse_store.root_synapse_cache.endswith(".synapseCache")
+
+    def test__purge_synapse_cache(self, synapse_store:SynapseStorage) -> None:
+        """Tests SynapseStorage._purge_synapse_cache"""
+        size_before_purge = check_synapse_cache_size(synapse_store.root_synapse_cache)
+        synapse_store._purge_synapse_cache(maximum_storage_allowed_cache_gb=0.000001, minute_buffer=0)
+        size_after_purge = check_synapse_cache_size(synapse_store.root_synapse_cache)
+        assert size_before_purge > size_after_purge
```

Comment:
I just ran this and saw that over 2000 files in my cache folder got deleted.... this is a bit scary.. 😦 I think here you should create a mock .synapseCache folder (and clean up after test is done) so that the tests won't actually modify the real .synapseCache folder stored in our root.